src/java/org/apache/cassandra/locator/SimpleStrategy.java

```diff
@@ -158,13 +161,4 @@ public Collection<String> recognizedOptions(ClusterMetadata metadata)
     {
         return Collections.singleton(REPLICATION_FACTOR);
     }
-
-    protected static void prepareOptions(Map<String, String> options, Map<String, String> previousOptions)
```

Comment:
This shouldn't be removed. It's only invoked by reflection (in `AbstractReplicationStrategy::prepareReplicationStrategyOptions`) so it appears unused. Restoring it fixes the 6 failing unit tests in the CI results.